src/core/lv_refr.c

```diff
@@ -288,7 +288,6 @@ void _lv_disp_refr_timer(lv_timer_t * tmr)
     REFR_TRACE("begin");
 
     uint32_t start = lv_tick_get();
```

Comment:
Do we need this variable?

Reply:
It just want a definite time point.
and record lv_tick_elaps cost. (YES it will be very small value).